recipes/syngap/requirements.txt

```diff
@@ -0,0 +1,9 @@
+more-itertools
```

Comment:
Thanks for the quick update. I've checked a few tools like scikit-image available on bioconda; pybedtools on bioconda; deap on the default channel. What was the reason to install them via pip? for the tools which are not available on conda, I'd put under the pip section in the meta.yml, no need to keep them in a separated file.

Reply:
I have updated the recipes, please review it again. Thanks!

Reply:
Great! Thanks for the quick update! We're almost there.